src/Components/CreateImageWizard/CreateImageWizard.js

```diff
@@ -474,6 +482,7 @@ const formStepHistory = (composeRequest, isBeta) => {
     }
 
     steps.push('image-name');
+    steps.push('image_description');
```

Comment:
Should this be `image_description` or `image-description`?

Reply:
chnaged